<!DOCTYPE html><html lang="en">
      
      
        
    
    
    
    

      
      
      <head>
        <meta charset="utf-8">
        <meta name="format-detection" content="telephone=no">  
        
        <title>Tools | Foundations of Computational and Systems Biology | Biology | MIT OpenCourseWare</title>
    <!-- Begin Automatic Metadata Insertion -->
    <meta content="7-91j-foundations-of-computational-and-systems-biology-spring-2014" name="WT.cg_n">
    <meta content="Tools" name="WT.cg_s">
    <meta content="This section contains links to tools and web resources that visitors may find helpful in their study of computational and systems biology." name="Description">
    <meta content="Burge, Christopher" name="Author">
    <meta content="Gifford, David" name="Author">
    <meta content="Fraenkel, Ernest" name="Author">
    <meta content="7.91J,7.91,20.490J,20.490,20.390J,20.390,7.36J,7.36,6.802J,6.802,6.874J,6.874,HST.506J,HST.506,computational biology,systems biology,bioinformatics,artificial intelligence,sequence analysis,proteomics,sequence alignment,protein folding,structure prediction,network modeling,phylogenetics,pairwise sequence comparisons,ncbi,blast,protein structure,dynamic programming,genome sequencing,DNA,RNA,x-ray crystallography,NMR,homologs,ab initio structure prediction,DNA microarrays,clustering,proteome,computational annotation,Computational Biology,Computation and Systems Biology,Computational Modeling and Simulation" name="keywords">
    <meta content="7.91J Foundations of Computational and Systems Biology | Tools" name="Search_Display">
    <meta content="Computational Biology" itemprop="about">
    <meta content="Computation and Systems Biology" itemprop="about">
    <meta content="Computational Modeling and Simulation" itemprop="about">
    <!-- End Automatic Metadata Insertion -->

	<link title="default" rel="stylesheet" type="text/css" href="../../common/styles/grid.css">
<link title="default" rel="stylesheet" type="text/css" href="../../common/styles/base.css">
<link title="default" rel="stylesheet" type="text/css" href="../../common/styles/menu.css">
<link title="default" rel="stylesheet" type="text/css" href="../../common/styles/jquery.bubblepopup.css">
<link title="default" rel="stylesheet" type="text/css" href="../../common/styles/search.css">
<link title="default" rel="stylesheet" type="text/css" href="../../common/styles/courses.css">
<link title="default" rel="stylesheet" type="text/css" href="../../common/styles/courses_new.css">
	<link href="../../common/xml/ocwcc.rdf" type="application/rdf+xml" rel="metadata">
	<link rel="canonical" href="https://ocw.mit.edu/courses/biology/7-91j-foundations-of-computational-and-systems-biology-spring-2014/tools/">
	<link rel="apple-touch-icon" href="../../common/images/apple-touch-icon.png">
	
	
      
	
	<script type="text/javascript" src="../../common/scripts/jquery.js"></script>
	<script type="text/javascript" src="../../common/scripts/ocw-media-utils-offline.js"></script>
	<script type="text/javascript" src="../../common/scripts/ocw-offline.js"></script>
	<script type="text/javascript" src="../../common/scripts/jquery.bubblepopup.min.js"></script>
	<script type="text/javascript" src="../../common/scripts/expandy.js"></script>
	<script type="text/javascript" src="https://ocw.mit.edu//z-na.amazon-adsystem.com/widgets/onejs?MarketPlace=US&amp;adInstanceId=70ac07d7-b28f-4071-816f-d292aba4849f"></script>
	<script type="text/javascript" src="../../common/scripts/bubble-popup-offline.js"></script>
	
	
	
    
    
    
    
    
    
    
      
       
		 

        
        
        

        
        
        
        
        
        
        
        
        
      </head>
    <body itemscope itemtype="http://schema.org/WebPage">
        
	

        <header id="top">
			<div id="grid">
				
				
					
<div id="portletwrapper-6f63772e746f70706f72746c65746d616e616765720a636f6e746578740a2f506c6f6e650a736974652d686561646572" class="portletWrapper kssattr-portlethash-6f63772e746f70706f72746c65746d616e616765720a636f6e746578740a2f506c6f6e650a736974652d686561646572">
<div class="portletStaticText portlet-static-site-header">
<!--googleoff: index-->
<div class="grid_6 alpha" id="banner"><a href="https://ocw.mit.edu/"><img src="../../common/images/ocw_mast.png" class="logo" alt="MIT OpenCourseWare, Massachusetts Institute of Technology"></a></div>
<div class="grid_6 omega" id="subscribe">
<aside class="module" aria-label="Connect with OCW">
<table class="social">
    <tbody>
        <tr>
            <td class="socialbutton"><a aria-label="Subscribe to the OCW Newsletter" href="https://ocw.mit.edu/subscribe/index.htm?utm_source=header"><img src="../../common/images/trans.gif" alt="An icon depicting an envelope.">Subscribe to the OCW Newsletter</a></td>
            <td>
<a aria-label="Facebook" href="https://facebook.com/mitocw"><img src="../../common/images/icon_fb.png" alt="Click to visit our Facebook page."></a>  <a aria-label="Instagram" href="https://www.instagram.com/mitocw/"><img src="https://ocw.mit.edu/images/icon_ig.png" alt="Click to visit our Instagram page."></a> <a aria-label="Twitter" href="https://twitter.com/mitocw"><img src="https://ocw.mit.edu/images/icon_twitter.png" alt="Click to visit our Twitter feed."></a><a aria-label="YouTube" href="https://www.youtube.com/mitocw" style="font-size: 12.208px;"><img src="https://ocw.mit.edu/images/icon_yt.png" alt="Click to visit our YouTube page."></a><span style="font-size: 12.208px;">   </span><a aria-label="Open Matters blog on WordPress" href="https://www.ocw-openmatters.org/" style="font-size: 12.208px;"><img src="https://ocw.mit.edu/images/icon_wp.png" alt="Click to visit our blog on WordPress."></a>
</td>
        </tr>
    </tbody>
</table>
</aside><nav aria-label="Help Links" class="helplinks">     <a aria-label="OCW Site Help" href="https://ocw.mit.edu/help">Help</a><span aria-hidden="true">|</span>     <a href="../../common/about/contactus.htm">Contact Us</a>   </nav>
</div>
<div class="clear"> </div>
<!--googleon: index-->
</div>

</div>

<div id="portletwrapper-6f63772e746f70706f72746c65746d616e616765720a636f6e746578740a2f506c6f6e650a6d6567612d6d656e75" class="portletWrapper kssattr-portlethash-6f63772e746f70706f72746c65746d616e616765720a636f6e746578740a2f506c6f6e650a6d6567612d6d656e75">
<div class="portletStaticText portlet-static-mega-menu"><div><nav id="mega" class="grid_8 alpha" aria-label="Site">
<ul id="menu" role="presentation">
    <li id="menu_home">
<a href="https://ocw.mit.edu/" aria-label="Homepage"><img src="../../common/images/top-nav_home.png" class="home_icon" alt="Click for site home page."></a><!-- Begin Home Item -->
</li>
    <!-- End Home Item -->
    <li id="drop_1" aria-label="Find Courses">
<a href="#" aria-hidden="true">FIND COURSES</a><!-- Begin 5 columns Item -->
    <div class="dropdown_5columns-a mega-courses">
    <div class="col_1a">
    <div class="row_1a">
<nav aria-labelledby="mm-find-courses-by">     <span id="mm-find-courses-by" class="nav" aria-hidden="true">Find courses by:</span>
    <ul class="find_by" role="presentation">
        <li><a href="https://ocw.mit.edu/courses/find-by-topic/">Topic</a></li>
        <li><a href="https://ocw.mit.edu/courses/find-by-number/">MIT Course Number</a></li>
        <li><a href="https://ocw.mit.edu/courses/find-by-department/">Department</a></li>
    </ul>
    </nav>     <nav aria-labelledby="mm-collections">     <span id="mm-collections" class="nav" aria-hidden="true">Collections</span>
    <ul role="presentation">
        <li><a href="https://ocw.mit.edu/courses/new-courses/">New Courses</a></li>
        <li><a href="https://ocw.mit.edu/courses/most-visited-courses/">Most Visited Courses</a></li>
        <li><a href="https://ocw.mit.edu/courses/ocw-scholar/">OCW Scholar Courses</a></li>
        <li><a href="https://ocw.mit.edu/courses/audio-video-courses/">Audio/Video Lectures</a></li>
        <li><a href="https://ocw.mit.edu/courses/online-textbooks/">Online Textbooks</a></li>
        <li><a href="https://ocw.mit.edu/resources/">Supplemental Resources</a></li>
        <li><a href="https://ocw.mit.edu/high-school/">OCW Highlights for High School</a></li>
        <li><a href="https://ocw.mit.edu/courses/mitx-related-courseware/">MITx &amp; Related OCW Courses</a></li>
        <li><a href="https://ocw.mit.edu/courses/mit-open-learning-library/">MIT Open Learning Library</a></li>
    </ul>
    </nav>     <nav class="col_1b" aria-labelledby="mm-translated-courses">     <span id="mm-translated-courses" class="nav" aria-hidden="true" style="line-height: 1.3;">Cross-Disciplinary Topic Lists</span>
    <ul role="presentation">
        <li><a href="https://ocw.mit.edu/courses/energy-courses">Energy</a></li>
        <li><a href="https://ocw.mit.edu/courses/entrepreneurship">Entrepreneurship</a></li>
        <li><a href="https://ocw.mit.edu/courses/environment-courses">Environment</a></li>
        <li><a href="https://ocw.mit.edu/courses/intro-programming">Introductory Programming</a></li>
        <li><a href="https://ocw.mit.edu/courses/life-sciences">Life Sciences</a></li>
        <li><a href="https://ocw.mit.edu/courses/transportation-courses">Transportation</a></li>
    </ul>
    </nav>
</div>
    <div class="row_1b"><nav aria-labelledby="mm-cross-disciplinary-topic-lists">     <span id="mm-cross-disciplinary-topic-lists" class="nav" aria-hidden="true">Translated Courses</span>
    <ul role="presentation">
        <li><a href="https://ocw.mit.edu/courses/translated-courses/traditional-chinese" aria-label="Traditional Chinese">繁體字 / Traditional Chinese</a></li>
        <li><a href="https://ocw.mit.edu/courses/translated-courses/turkish" aria-label="Turkish">Türkçe / Turkish</a></li>
        <li><a href="https://ocw.mit.edu/courses/translated-courses/korean" aria-label="Korean">(비디오)한국 / Korean</a></li>
    </ul>
    </nav></div>
    </div>
    </div>
    </li>
    <li id="drop_2">
<a href="#" aria-label="For Educators">For Educators</a>
    <div class="dropdown_1column-a" style="width: 270px;"><nav aria-labelledby="mm-find-courses-by">
    <ul role="presentation">
        <li><a href="https://ocw.mit.edu/educator/chalk-radio-podcast">Chalk Radio Podcast</a></li>
        <li><a href="https://ocw.mit.edu/educator/">OCW Educator Portal</a></li>
    </ul>
    <ul role="presentation">
        <li><a href="https://ocw.mit.edu/courses/instructor-insights/">Instructor Insights by Department</a></li>
        <li><a href="https://openlearning.mit.edu/campus/digital-innovations/">Residential Digital Innovations </a></li>
    </ul>
    <ul role="presentation">
        <li><a href="https://ocw.mit.edu/high-school/">OCW Highlights for High School</a></li>
    </ul>
    <ul role="presentation">
        <li><a href="https://ocw.mit.edu/educator/additional-resources/">Additional Resources</a></li>
    </ul>
    </nav></div>
    </li>
    <li id="drop_3">
<a href="#" aria-hidden="true">Give Now</a>
    <div class="dropdown_1column-a"><nav class="col_1" aria-label="Donate">
    <ul role="presentation">
        <li><a href="https://ocw.mit.edu/give/">Make a Donation</a></li>
        <li><a href="https://ocw.mit.edu/give/why-give/">Why Give?</a></li>
        <li><a href="https://ocw.mit.edu/give/our-supporters/">Our Supporters</a></li>
        <li><a href="https://ocw.mit.edu/give/other-ways-to-contribute/">Other Ways to Contribute</a></li>
        <li><a href="https://ocw.mit.edu/support/">Become a Corporate Sponsor</a></li>
    </ul>
    </nav></div>
    </li>
    <li id="drop_4">
<a href="#" aria-hidden="true">About</a>
    <div class="dropdown_1column-a"><nav class="col_1" aria-label="About">
    <ul role="presentation">
        <li><a href="https://ocw.mit.edu/about/">About MIT OpenCourseWare</a></li>
        <li><a href="https://ocw.mit.edu/about/site-statistics/">Site Statistics</a></li>
        <li><a href="https://ocw.mit.edu/about/ocw-stories/">OCW Stories</a></li>
        <li><a href="https://ocw.mit.edu/about/newsletter/">Newsletter</a></li>
        <li><a href="https://chalk-radio.simplecast.com/">Chalk Radio Podcast</a></li>
        <li><a href="https://www.ocw-openmatters.org/">Open Matters Blog</a></li>
    </ul>
    </nav></div>
    </li>
</ul>
</nav></div></div>

</div>





<!--googleoff: index-->
<script>
  (function() {
	var cx = '012626166551961672889:owjdpuboktq';
	var gcse = document.createElement('script');
	gcse.type = 'text/javascript';
	gcse.async = true;
	gcse.src = 'https://cse.google.com/cse.js?cx=' + cx;
	var s = document.getElementsByTagName('script')[0];
	s.parentNode.insertBefore(gcse, s);
  })();
  window.onload = function(){
	document.getElementById('gsc-i-id1').placeholder = 'Search';	
  };

$(document).ready(function(){

$('.advanceSearch a').keydown(function(event){showSearchTips($(this),event);})
$('#searchTipsModal').keydown(function(event){showSearchTips($(this),event);})

function showSearchTips(obj,evt) {
// if pressed enter key
	if ( evt.which == 13 || evt.which == 32) {
		showModal();
		ga('send', 'pageview', "AdvanceSearch");
		$(".advanceSearch a").attr('aria-expanded', 'true');
		$('.searchTipsModal').focus();
		evt.preventDefault();
		}
	if ( evt.which == 27 ) {
		hideModal();
		$(".advanceSearch a").attr('aria-expanded', 'false');
		var modal = document.getElementById('searchTipsModal');
		modal.style.display = "none";
		$(".advanceSearch a").focus();
		evt.preventDefault();
		}
}

});
function showModal(){
	var modal = document.getElementById('searchTipsModal');
	modal.style.display = "block";
	ga('send', 'pageview', "AdvanceSearch");
	document.getElementById("searchTipsBtn").setAttribute("aria-expanded", true);
	window.setTimeout(function () {
    document.getElementById('searchTipsModal').focus(); }, 0);
}

function hideModal(){
	var modal = document.getElementById('searchTipsModal');
	modal.style.display = "none";
	document.getElementById("searchTipsBtn").setAttribute("aria-expanded", false);
}
</script>
<div id="search" role="search" class="grid_4 omega">
    	<table class="search">
				<tbody>
					<tr>
						<td><div class="searchboxheader"><searchbox-only resultsurl="/search/ocwsearch.htm"></searchbox-only></div></td>
						<td>
							<div class="advanceSearch">
								<a id="searchTipsBtn" onclick="showModal();" href="#" role="button" aria-label="search tips" aria-expanded="false" aria-describedby="searchtips">Search Tips</a>

								<!-- The Modal -->
								<div id="searchTipsModal" class="modal" tabindex="-1">
								  <!-- Modal content -->
									<div class="modal-content">
										<div class="modal-body">
											<button class="close" onclick="hideModal();" aria-label="close">X</button>
											<span>
												<b>Exclude words from your search</b>
												<br>Put - in front of a word you want to leave out. For example, jaguar speed -car
												<br><br>
												<b>Search for an exact match</b>
												<br>Put a word or phrase inside quotes. For example, "tallest building".
												<br><br>
												<b>Search for wildcards or unknown words</b>
												<br>Put a * in your word or phrase where you want to leave a placeholder. For example, "largest * in the world".
												<br><br>
												<b>Search within a range of numbers</b>
												<br>Put .. between two numbers. For example, camera $50..$100.
												<br><br>
												<b>Combine searches</b>
												<br>Put "OR" between each search query. For example,  marathon OR race.
												<br><br>
											</span>
										</div>
								  </div>
								</div>
							</div>
						</td>
					</tr>
				</tbody>
		</table>
</div>
<div class="clear"></div>
<!--googleon: index-->
<!-- *end header* -->

				
				
			</div>
<!-- top grid end -->
		</header><!-- top end -->
			
		<div id="center">
         <div id="grid">
		         <div id="left-section">
                 <nav aria-label="Breadcrumb" id="breadcrumb">
                      <p>

    <a href="https://ocw.mit.edu/">Home</a>
    
        »
        
    
    
        
            <a href="https://ocw.mit.edu/courses">Courses</a>
            
                »
                
            
            
         
    
    
        
            <a href="https://ocw.mit.edu/courses/biology">Biology</a>
            
                »
                
            
            
         
    
    
        
            <a href="../../contents/index.htm">Foundations of Computational and Systems Biology</a>
            
                »
                
            
            
         
    
    
        
            
            
            Tools
         
    
</p>

                 </nav>
                 <div class="clear"></div>
                   <div id="section_title">
           			     <h1 class="title" itemprop="name" property="dct:title">
        <span class="" id="parent-fieldname-title">
            Tools
        </span>
    </h1>
             	     </div>
             	     <div class="clear"></div>
             	     <div id="course_wrapper_section">
           	     	 <nav id="course_nav" aria-label="Course">
               			 



<script language="javascript" type="text/javascript">
function toggleMenu(objID) {
  if (!document.getElementById) return;
  var ob = document.getElementById(objID);
  ob.className = (ob.className == 'selected')?'': 'selected';
}
function toggleClass(id)
{
  var divtoggleClass= document.getElementById(id);
  divtoggleClass.className = (divtoggleClass.className == 'mO')?'mC': 'mO';
  return false;
}
function changeAlt(id)
{
  id.alt = (id.alt == 'Expand Menu')?'Collapse Menu' : 'Expand Menu';
  id.title = (id.title == 'Expand Menu')?'Collapse Menu' : 'Expand Menu';
}
</script>
<!--Left Nav Starts -->


	<ul>			  
	
	    	
	    	    <li class="">
			   			<a href="../../contents/index.htm">
		                  Course Home  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../contents/syllabus/index.htm">
		                  Syllabus  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../contents/calendar/index.htm">
		                  Calendar  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../contents/instructor-insights/index.htm">
		                  Instructor Insights  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../contents/readings/index.htm">
		                  Readings  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../contents/lecture-slides/index.htm">
		                  Lecture Slides  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../contents/video-lectures/index.htm">
		                  Video Lectures  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../contents/recitations/index.htm">
		                  Recitations  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../contents/assignments/index.htm">
		                  Assignments  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="">
			   			<a href="../../contents/project/index.htm">
		                  Project  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    <li class="selected">
			   			<a href="../../contents/tools/index.htm">
		                  Tools  			                
	                    </a>
		        </li>
		    
         	
	
	
	    	
	    	    
		    
         	
	<!--second tal block close-->  
	
</ul>


<!--Left Nav Ends -->









           		     </nav>
           		     <main id="course_inner_section" aria-labelledby="section_title">
           		     	
	       		     	<h2 class="subhead">Computational Biology Tools and Resources</h2> <div class="maintabletemplate"><table class="tablewidth100" summary="See table caption for summary.">     <caption class="invisible">Course tools.</caption>       <thead>         <tr>             <th scope="col">TOOLS</th>             <th scope="col">DESCRIPTIONS</th>             <th scope="col">REFERENCES</th>         </tr>     </thead>          <tbody>         <tr class="row">             <td><a href="http://blast.ncbi.nlm.nih.gov/Blast.cgi">BLAST</a></td>             <td>Sequence alignments provide a powerful way to compare novel sequences with previously characterized genes. Both functional and evolutionary information can be inferred from well designed queries and alignments. The Basic Local Alignment Search Tool (BLAST) provides a method for rapid searching of nucleotide and protein databases. It is a sequence comparison algorithm optimized for speed used to search sequence databases for optimal local alignments to a query.</td>             <td>Altschul, S. F., W. Gish, et al. "<a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2">Basic Local Alignment Search Tool</a>." <em>Journal of Molecular Biology</em> 215, no. 3 (1990): 403–10.</td>         </tr>         <tr class="alt-row">             <td><a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=Retrieve&amp;list_uids=11125103">BIND</a></td>             <td>The Biomolecular Interaction Network Database (BIND) is a collection of records documenting molecular interactions. The contents of BIND include high-throughput data submissions and hand-curated information gathered from the scientific literature.</td>             <td>Bader, G. D., D. Betel, et al. "<a href="http://dx.doi.org/10.1093/nar/gkg056">BIND: The Biomolecular Interaction Network Database</a>." <em>Nucleic Acids Research</em> 31, no. 1 (2003): 248–50.</td>         </tr>         <tr class="row">             <td><a href="http://workbench.sdsc.edu/">Biology WorkBench</a></td>             <td>The Biology WorkBench is a web-based tool for biologists developed by the San Diego Supercomputer Center at the University of California San Diego. The WorkBench allows biologists to search many popular protein and nucleic acid sequence databases. Database searching is integrated with access to a wide variety of analysis and modeling tools, all within a point and click interface that eliminates file format compatibility problems.</td>             <td> </td>         </tr>         <tr class="alt-row">             <td><a href="http://www.ebi.ac.uk/Tools/msa/">ClustalW</a></td>             <td>ClustalW is a general purpose multiple sequence alignment program for DNA or proteins. It produces biologically meaningful multiple sequence alignments of divergent sequences.</td>             <td>Thompson, J. D., D. G. Higgins, et al. "<a href="http://dx.doi.org/10.1093/nar/22.22.4673">CLUSTAL W: Improving the Sensitivity of Progressive Multiple Sequence Alignment through Sequence Weighting, Position-specific Gap Penalties and Weight Matrix Choice</a>." <em>Nucleic Acids Research</em> 22, no. 22 (1994): 4673–80.</td>         </tr>         <tr class="row">             <td><a href="http://www.ebi.ac.uk/msd-srv/ssm/">DALI</a></td>             <td>DALI stands for Distance mAtrix aLIgnment. The Dali server is an automatic service for the comparison of protein structure in 3D. You send the coordinates of a query structure and receive a multiple structure alignment in return. You can submit your coordinates either by electronic mail or interactively from the World Wide Web.</td>             <td>Holm, L., and C. Sander. "<a href="http://dx.doi.org/10.1126/science.273.5275.595">Mapping the Protein Universe</a>." <em>Science</em> 273, no. 5275 (1996): 595–602.</td>         </tr>         <tr class="alt-row">             <td>
<p><a href="http://spdbv.vital-it.ch/">Deep View Swiss-Pdb Viewer</a></p>             <p><a href="http://spdbv.vital-it.ch/TheMolecularLevel/SPVTut/">Tutorial For Deep View (Swiss-PdbViewer)</a></p>
</td>             <td>The Deep View Swiss-PdbViewer is a software application with a user friendly interface that allows one to analyze several proteins at the same time. The proteins can be superimposed in order to deduce structural alignments and compare their active sites or any other relevant parts. Amino acid mutations, H-bonds, angles, and distances between atoms are easy to obtain thanks to the intuitive graphic and menu interface.</td>             <td>Guex, N., and M. C. Peitsch. "<a href="http://dx.doi.org/10.1002/elps.1150181505">SWISS-MODEL and the Swiss-PdbViewer: An Environment for Comparative Protein Modeling</a>." <em>Electrophoresis</em> 18, no. 15 (1997): 2714–23.</td>         </tr>         <tr class="row">             <td>
<a href="http://dip.doe-mbi.ucla.edu/dip/Main.cgi">DIP</a><sup>TM</sup>
</td>             <td>DIP<sup>TM</sup> stands for Database of Interacting Proteins. The DIP<sup>TM</sup> database catalogs experimentally determined interactions between proteins. It combines information from a variety of sources to create a single, consistent set of protein-protein interactions.</td>             <td>Salwinski L., C. S. Miller, et al. "<a href="http://dx.doi.org/10.1093/nar/gkh086">The Database of Interacting Proteins: 2004 Update</a>." <em>Nucleic Acids Research</em> 32, no. 1 (2004): D449–51.</td>         </tr>         <tr class="alt-row">             <td>Dot Matrix</td>             <td>Dot or matrix plots provide an easy and powerful means of sequence analysis for searching out regions of similarity in two sequences and repeats within a single sequence.</td>             <td>
<p>Maizel, J. V., and R. P. Lenk. "<a href="http://dx.doi.org/10.1073/pnas.78.12.7665">Enhanced Graphic Matrix Analysis of Nucleic Acid and Protein Sequences</a>." <em>Proceedings of the National Academy Sciences</em> 78, no. 12 (1981): 7665–9.</p>             <p>Pustell, J., and F. C. Kafatos. "<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC321127/">A High Speed, High Capacity Homology Matrix: Zooming through SV40 and Polyoma</a>." <em>Nucleic Acids Research</em> 10, no. 15 (1982): 4765–82.</p>             <p>Quigley, G. J., L. Gehrke, et al. "<a href="http://dx.doi.org/10.1093/nar/12.1Part1.347">Computer-aided Nucleic Acid Secondary Structure Modeling Incorporating Enzymatic Digestion Data</a>." <em>Nucleic Acids Research</em> 12, no. 1 (1984): 347–66.</p>
</td>         </tr>         <tr class="row">             <td>
<p><a href="http://www.ncbi.nlm.nih.gov/gquery/">Entrez</a></p>             <p><a href="http://www.ncbi.nlm.nih.gov/">NCBI</a></p>
</td>             <td>Entrez is a retrieval system designed for searching several linked databases at NCBI (National Center for Biotechnology Information).</td>             <td> </td>         </tr>         <tr class="alt-row">             <td><a href="http://hollywood.mit.edu/GENSCAN.html">GENSCAN</a></td>             <td>GENSCAN predicts the locations and exon-intron structures of genes in genomic sequences from a variety of organisms. GENSCAN was developed by Prof. Chris Burge while he was in the research group of Samuel Karlin, Department of Mathematics, Stanford University.</td>             <td>Burge, C., and S. Karlin. "<a href="http://dx.doi.org/10.1006/jmbi.1997.0951">Prediction of Complete Gene Structures in Human Genomic DNA</a>." <em>Journal of Molecular Biology</em> 268, no. 1 (1997): 78–94.</td>         </tr>         <tr class="row">             <td><a href="https://omictools.com/the-gibbs-motif-sampler-tool">Gibbs Motif Sampler</a></td>             <td>The Gibbs Motif Sampler will allow you to identify motifs, conserved regions, in DNA or protein sequences.</td>             <td>This software was developed by Eric C. Rouchka and Bill Thompson based on work by C. E. Lawrence, J. S. Liu, A. F. Neuwald and others as part of the Bayesian Bioinformatics Program at the Biometrics Laboratory of Wadsworth Center.</td>         </tr>         <tr class="alt-row">             <td><a href="http://meme-suite.org/">MEME</a></td>             <td>Discover motifs (highly conserved regions) in groups of related DNA or protein sequences using MEME.</td>             <td>Bailey, Timothy L., and Charles Elkan. "<a href="http://www.ncbi.nlm.nih.gov/pubmed/7584402">Fitting a Mixture Model by Expectation Maximization to Discover Motifs in Biopolymers</a>." <em>Proceedings of the Second International Conference on Intelligent Systems for Molecular Biology</em> 2 (1994): 28–36.</td>         </tr>         <tr class="row">             <td>MODBASE</td>             <td>MODBASE is a database of annotated comparative protein structure models and associated resources.</td>             <td>Pieper U., N. Eswar, et al. "<a href="http://dx.doi.org/10.1093/nar/gkj059">MODBASE: A Database of Annotated Comparative Protein Structure Models, and Associated Resources</a>." <em>Nucleic Acids Research</em> 34, no. 1 (2004): D291–5.</td>         </tr>         <tr class="alt-row">             <td><p><a href="http://www.pdb.org/pdb/home/home.do">PDB<br>             </a></p></td>             <td>The Protein Data Bank (PDB) is the single worldwide repository for the processing and distribution of 3-D biological macromolecular structure data.</td>             <td>Berman, H. M., J. Westbrook, et al. "<a href="http://dx.doi.org/10.1093/nar/28.1.235">The Protein Data Bank</a>." <em>Nucleic Acids Research</em> 28, no. 1 (2000): 235–42.</td>         </tr>         <tr class="row">             <td><a href="http://evolution.genetics.washington.edu/phylip.html">PHYLIP</a></td>             <td>PHYLIP (the PHYLogeny Inference Package) is a package of programs for inferring phylogenies (evolutionary trees). It is available free over the Internet, and written to work on as many different kinds of computer systems as possible.</td>             <td> </td>         </tr>         <tr class="alt-row">             <td><a href="http://www.ncbi.nlm.nih.gov/">NCBI</a></td>             <td>Established in 1988 as a national resource for molecular biology information, NCBI (National Center for Biotechnology Information) creates public databases, conducts research in computational biology, develops software tools for analyzing genome data, and disseminates biomedical information—all for the better understanding of molecular processes affecting human health and disease. This web site provides access to a myriad of biological sequence databases, structural databases, bioinformatics tools, and literature search tools.</td>             <td> </td>         </tr>         <tr class="row">             <td><a href="https://www.python.org/">Python Scripting Language</a></td>             <td>A scripting language widely used, along with PERL, for bioinformatics and computational biology.</td>             <td> </td>         </tr>         <tr class="alt-row">             <td><a href="http://openrasmol.org/">RasMol</a></td>             <td>RasMol is a molecular graphics program intended for the visualization of proteins, nucleic acids and small molecules. The program reads in molecular co-ordinate files and interactively displays the molecule on the screen in a variety of representations and colour schemes.</td>             <td>Sayle R., and E. James Milner-White. "<a href="http://dx.doi.org/10.1016/S0968-0004(00)89080-5">RasMol: Biomolecular Graphics for All</a>." <em>Trends in Biochemical Sciences</em> 20, no. 9 (1995): 374–6.</td>         </tr>         <tr class="row">             <td><a href="http://scansite3.mit.edu/#home">Scansite</a></td>             <td>Scansite searches for motifs within proteins that are likely to be phosphorylated by specific protein kinases or bind to domains such as SH2 domains, 14-3-3 domains or PDZ domains.</td>             <td>
<p>Songyang, Z., S. Blechner, et al. "<a href="http://dx.doi.org/10.1016/S0960-9822(00)00221-9">Use of an Oriented Peptide Library to Determine the Optimal Substrates of Protein Kinases</a>." <em>Current Biology</em> 4, no. 11 (1994): 973–82.</p>             <p>Yaffe, M. B., G. G. Leparc, et al. "<a href="http://dx.doi.org/10.1038/86737">A Motif-based Profile Scanning Approach for Genome-wide Prediction of Signaling Pathways</a>." <em>Nature Biotechnology</em> 19, no. 4 (2001): 348–53.</p>             <p>Obenauer, J. C., L. C. Cantley, et al. "<a href="http://dx.doi.org/10.1093/nar/gkg584">Scansite 2.0: Proteome-wide Prediction of Cell Signaling Interactions using Short Sequence Motifs</a>." <em>Nucleic Acids Research</em> 31, no. 13 (2003): 3635–41.</p>
</td>         </tr>         <tr class="alt-row">             <td><a href="http://scop.mrc-lmb.cam.ac.uk/scop/">SCOP</a></td>             <td>Nearly all proteins have structural similarities with other proteins and, in some of these cases, share a common evolutionary origin. The Structural Classification Of Proteins (SCOP) database, created by manual inspection and abetted by a battery of automated methods, aims to provide a detailed and comprehensive description of the structural and evolutionary relationships between all proteins whose structure is known. As such, it provides a broad survey of all known protein folds, detailed information about the close relatives of any particular protein, and a framework for future research and classification.</td>             <td>Murzin, A. G., S. E. Brenner, et al. "<a href="http://dx.doi.org/10.1016/S0022-2836(05)80134-2">SCOP: A Structural Classification of Proteins Database for the Investigation of Sequences and Structures</a>." <em>Journal of Molecular Biology</em> 247, no. 4 (1995): 536–40.</td>         </tr>         <tr class="row">             <td><a href="http://www.cbs.dtu.dk/services/TMHMM/">TMHMM</a></td>             <td>This software tool was developed by the Center for Biological Sequence Analysis at the Technical University of Denmark and is used to predict transmembrane helices in protein sequences.</td>             <td>Krogh, A., B. Larsson, B., et al. "<a href="http://dx.doi.org/10.1006/jmbi.2000.4315">Predicting Transmembrane Protein Topology with a Hidden Markov Model: Application to Complete Genomes</a>." <em>Journal of Molecular Biology</em> 305, no. 3 (2001): 567–80.</td>         </tr>              </tbody> </table></div>
	                    
    
    
    



	        			
	                    
	                    
	                    
	                    

            		 </main><!--Course_inner_section tag close -->
           		     <div class="clear"></div>
 				 </div>
<!--Course_wrapper tag close -->
 			 </div>
<!--left tag close -->
 			
         </div>
<!--grid tag close -->
      </div>
		
		<footer id="bottom">
			<div id="grid">
				
<div id="portletwrapper-6f63772e626f74746f6d706f72746c65746d616e616765720a636f6e746578740a2f506c6f6e650a736974652d666f6f746572" class="portletWrapper kssattr-portlethash-6f63772e626f74746f6d706f72746c65746d616e616765720a636f6e746578740a2f506c6f6e650a736974652d666f6f746572">
<div class="portletStaticText portlet-static-site-footer"><div id="footer">
<nav aria-label="Footer">     <nav id="foot-c1" class="grid_2 alpha" aria-labelledby="f-find-courses">       <span class="footer" id="f-find-courses" aria-hidden="true">Find Courses</span>
<ul class="foot-bullet" role="presentation">
    <li><a href="https://ocw.mit.edu/courses/find-by-topic/">Find by Topic</a></li>
    <li><a href="https://ocw.mit.edu/courses/find-by-number/">Find by Course Number</a></li>
    <li><a href="https://ocw.mit.edu/courses/find-by-department/">Find by Department</a></li>
    <li><a href="https://ocw.mit.edu/courses/new-courses/">New Courses</a></li>
    <li><a href="https://ocw.mit.edu/courses/most-visited-courses/">Most Visited Courses</a></li>
    <li><a href="https://ocw.mit.edu/courses/ocw-scholar/">OCW Scholar Courses</a></li>
    <li><a href="https://ocw.mit.edu/courses/audio-video-courses/">Audio/Video Courses</a></li>
    <li><a href="https://ocw.mit.edu/courses/online-textbooks/">Online Textbooks</a></li>
    <li><a href="https://ocw.mit.edu/courses/instructor-insights/">Instructor Insights</a></li>
    <li><a href="https://ocw.mit.edu/resources/">Supplemental Resources</a></li>
    <li><a href="https://ocw.mit.edu/courses/mitx-related-courseware/">MITx &amp; Related OCW Courses</a></li>
    <li><a href="https://ocw.mit.edu/courses/mit-open-learning-library/">MIT Open Learning Library</a></li>
    <li><a href="https://ocw.mit.edu/courses/translated-courses/">Translated Courses</a></li>
</ul>
</nav>
<div id="foot-c2" class="grid_2"><nav aria-labelledby="f-for-educators">         <span id="f-for-educators" class="footer" aria-hidden="true">For Educators</span>
<ul class="foot-bullet" role="presentation">
    <li><a href="https://chalk-radio.simplecast.com/">Chalk Radio Podcast</a></li>
    <li><a href="https://ocw.mit.edu/educator/">OCW Educator Portal </a></li>
    <li><a href="https://ocw.mit.edu/courses/instructor-insights/">Instructor Insights by Department</a></li>
    <li><a href="https://openlearning.mit.edu/campus/digital-innovations/" aria-label="External Link: Residential Digital Innovations">Residential Digital Innovations </a></li>
    <li><a href="https://ocw.mit.edu/high-school/">OCW Highlights for High School</a></li>
    <li><a href="https://ocw.mit.edu/educator/additional-resources/">Additional Resources</a></li>
</ul>
</nav></div>
<nav class="grid_2" id="foot-c3" aria-labelledby="f-donate">       <span id="f-donate" class="footer" aria-hidden="true">Give Now</span>
<ul class="foot-bullet" role="presentation">
    <li><a href="https://ocw.mit.edu/give/">Make a Donation</a></li>
    <li><a href="https://ocw.mit.edu/give/why-give/">Why Give?</a></li>
    <li><a href="https://ocw.mit.edu/give/our-supporters/">Our Supporters</a></li>
    <li><a href="https://ocw.mit.edu/give/other-ways-to-contribute/">Other Ways to Contribute</a></li>
    <li><a href="https://ocw.mit.edu/support/">Become a Corporate Sponsor</a></li>
</ul>
</nav>
<div class="grid_2" id="foot-c4">
<nav aria-labelledby="f-about">         <span id="f-about" class="footer" aria-hidden="true">About</span>
<ul class="foot-bullet" role="presentation">
    <li><a href="https://ocw.mit.edu/about/">About OpenCourseWare</a></li>
    <li><a href="https://ocw.mit.edu/about/site-statistics/">Site Statistics</a></li>
    <li><a href="https://ocw.mit.edu/about/ocw-stories/">OCW Stories</a></li>
    <li><a href="https://ocw.mit.edu/about/newsletter/">Newsletter</a></li>
    <li><a href="https://www.ocw-openmatters.org/">Open Matters Blog</a></li>
</ul>
</nav><!--about-->       <nav aria-labelledby="f-tools">         <span id="f-tools" class="footer" aria-hidden="true">Tools</span>
<ul class="foot-bullet" role="presentation">
    <li><a href="https://ocw.mit.edu/help/">Help &amp; FAQs</a></li>
    <li><a href="https://ocw.mit.edu/about/contactus">Contact Us</a></li>
    <li><a href="https://accessibility.mit.edu/" target="_blank">Accessibility</a></li>
    <li><a href="https://ocw.mit.edu/help/site-map/">Site Map</a></li>
    <li><a href="../../common/terms/index.htm">Privacy &amp; Terms of Use</a></li>
    <li><a href="https://ocw.mit.edu/help/rss/">RSS Feeds</a></li>
</ul>
</nav><!--tools-->
</div>
</nav> <aside style="min-height: 289px;" aria-labelledby="f-our-corporate-supporters" class="grid_4 omega" id="foot-c5">           <span aria-hidden="true" class="footer" id="f-our-corporate-supporters">Our Corporate Supporters</span>           <!-- HOME_CORP_LOGO_1 -->
<div id="div-gpt-ad-1388181177156-0" class="sponsors_google_ads_even"><script type="text/javascript">
              googletag.cmd.push(function() { googletag.display('div-gpt-ad-1388181177156-0'); });
            </script></div>
<!-- HOME_CORP_LOGO_2 -->
<div id="div-gpt-ad-1388181177156-1" class="sponsors_google_ads_odd"><script type="text/javascript">
              googletag.cmd.push(function() { googletag.display('div-gpt-ad-1388181177156-1'); });
            </script></div>
<!-- HOME_CORP_LOGO_3 -->
<div id="div-gpt-ad-1388181177156-2" class="sponsors_google_ads_even"><script type="text/javascript">
              googletag.cmd.push(function() { googletag.display('div-gpt-ad-1388181177156-2'); });
            </script></div>
<!-- HOME_CORP_LOGO_4 -->
<div id="div-gpt-ad-1388181177156-3" class="sponsors_google_ads_odd"><script type="text/javascript">
              googletag.cmd.push(function() { googletag.display('div-gpt-ad-1388181177156-3'); });
            </script></div>
<!-- HOME_CORP_LOGO_5 -->
<div id="div-gpt-ad-1388181177156-4" class="sponsors_google_ads_even"><script type="text/javascript">
              googletag.cmd.push(function() { googletag.display('div-gpt-ad-1388181177156-4'); });
              </script></div>
<!-- HOME_CORP_LOGO_6 -->
<div id="div-gpt-ad-1388181177156-5" class="sponsors_google_ads_odd"><script type="text/javascript">
              googletag.cmd.push(function() { googletag.display('div-gpt-ad-1388181177156-5'); });
              </script></div>
</aside>
<div class="grid_12 alpha omega" itemprop="publisher" itemscope="" itemtype="http://schema.org/CollegeOrUniversity">
<h4 class="footer" style="border-top: thin solid #d5c9ba; padding-top: 10px; margin-bottom: 10px;">About <span itemprop="name">MIT OpenCourseWare</span>
</h4>
<p style="color: #999; font-size: 1em; line-height: 1.5em; margin-top: 10px;" itemprop="description">MIT OpenCourseWare is an online publication of materials from over 2,500 MIT courses, freely sharing knowledge with learners and educators around the world. <a href="https://ocw.mit.edu/about/">Learn more »</a></p>
</div>
<div id="foot-copy" class="grid_12 alpha omega" style="border-top: none;">
<a href="http://web.mit.edu"><img src="../../common/images/logo_mit.png" alt="Massachusetts Institute of Technology logo and name." style="width: 195; height: 44;"></a><a href="https://openlearning.mit.edu/"><img src="https://ocw.mit.edu/images/mitol_logo.png" alt="MIT Open Learning logo and name." style="width: 265; height: 50; vertical-align: top; padding-left:30px;"></a><a href="https://www.oeglobal.org/"><img src="https://ocw.mit.edu/images/oeg_logo.gif" alt="Open Education Consortium logo." style="width: 219px; height: 59px; vertical-align: top; padding-left:20px;"></a><a rel="license" itemprop="useRightsUrl" href="http://creativecommons.org/licenses/by-nc-sa/4.0/"><img src="../../common/images/cc_by-nc-sa.png" alt="Creative Commons logo with terms BY-NC-SA." style="width: 126px; height: 44px; margin-right: 0; padding-left: 20px;"></a>
<p class="copyright">© 2001–2018<br>
Massachusetts Institute of Technology</p>
<p style="font-size: 0.9em; margin-bottom: 15px;">Your use of the MIT OpenCourseWare site and materials is subject to our <a href="https://creativecommons.org/licenses/by-nc-sa/4.0/" rel="license">Creative Commons License</a> and other <a href="../../common/terms/index.htm" rel="cc:morePermissions">terms of use</a>.</p>
</div>
</div></div>

</div>





                
			</div> <!-- bottom grid end -->
		</footer><!-- footer bottom end -->


   </body>
 </html>
